tdigest/test/tdigest_test.cpp

```diff
@@ -54,8 +54,9 @@ TEST_CASE("one value", "[tdigest]") {
 
 TEST_CASE("many values", "[tdigest]") {
   const size_t n = 10000;
-  tdigest_double td(100);
+  tdigest_double td;
   for (size_t i = 0; i < n; ++i) td.update(i);
+//  std::cout << td.to_string(true);
```

Comment:
not a blocker but some extra comments left in it seems

Reply:
these are tests, and I uncomment this sometimes to check. not a big deal. we can kill this later